test/integration/test_models.py

```diff
@@ -68,3 +68,51 @@ def test_set_medians(conn: Connector, monkeypatch):
 
     assert instance.median_30_day_failed == 10.5
     assert instance.median_30_day_passed == 155
+
+
+def test_upsert_qualitycheck(conn: Connector):
```

Comment:
Isn't this rather unittest for the Connector's `upsert` method? Perhaps would be better to move it to _unit_ dir.

Reply:
I think its integration test. yes, it is for connector, not the model. can create new file

Reply:
thanks